src/lib/layout/containerHeader.svelte

```diff
@@ -124,13 +125,19 @@
         {#if isCloud && isLimited}
             <DropList bind:show={showDropdown} width="16">
                 {#if hasProjectLimitation}
-                    <Pill button on:click={() => (showDropdown = !showDropdown)}>
-                        <span class="icon-info"></span>{total}/{limit} created
-                    </Pill>
+                    <Badge
+                        variant="secondary"
+                        color="pink-2"
+                        on:click={() => (showDropdown = !showDropdown)}
+                        style="cursor:pointer;"
+                        content={`ⓘ ${total}/${limit} created`} />
```

Comment:
we can remove the `ⓘ` here. `Badge` accepts a `start` slot so we can add an `Icon` component.